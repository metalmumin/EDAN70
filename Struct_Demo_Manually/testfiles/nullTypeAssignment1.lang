struct A {
	int d;
}

a.d = 34;
