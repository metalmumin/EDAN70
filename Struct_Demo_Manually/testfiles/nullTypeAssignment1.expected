Error at line 5: Assignment to type 'null' with type 'int' is illegal.